<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeCAMThresholdsWidget</class>
 <widget class="QWidget" name="EdgeCAMThresholdsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>崩边</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
    <property name="text">
     <string>毫米</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>崩边面积阈值：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="white_length_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
    <property name="text">
     <string>0.0 ~ 1.0</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="white_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
    <property name="text">
     <string>平方毫米</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="white_area_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>崩边置信概率：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>崩边长度阈值：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>亮斑</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
    <property name="text">
     <string>毫米</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>亮斑长度阈值：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
    <property name="text">
     <string>0.0 ~ 1.0</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="black_length_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>亮斑置信概率：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="black_area_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
    <property name="text">
     <string>平方毫米</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="black_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>亮斑面积阈值：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>分类筛选</string>
   </property>
   <widget class="QCheckBox" name="white_checkbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>崩边</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_checkbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>亮斑</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
